src/challenges/challengeOne/5.Modal/Modal.test.tsx

```diff
@@ -1,8 +1,54 @@
-// import React from "react";
-// import { render, screen } from "@testing-library/react";
-// import userEvent from "@testing-library/user-event";
-// import { Modal } from "./Modal";
-
-// describe("Modal", () => {
-//   it("renders...", () => {});
-// });
+import React from "react";
+import { render, screen } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+import { Modal, ModalProps } from "./Modal";
+
+describe("Modal", () => {
+  const onClose = jest.fn();
+  const onConfirm = jest.fn();
+
+  const renderComponent = (additionalProps?: Partial<ModalProps>) => {
+    const defaultProps: ModalProps = {
+      isOpen: true,
```

Comment:
You want to test it renders when isOpen is true and doesn't when isOpen is false